modules/apps/web-experience/export-import/export-import-resources-importer/src/main/java/com/liferay/exportimport/resources/importer/internal/messaging/ResourcesImporterHotDeployMessageListener.java

```diff
@@ -73,6 +78,16 @@
 
 	@Activate
 	protected void activate(final BundleContext bundleContext) {
+		_bundleContext = bundleContext;
+
+		_messageQueue = new LinkedList<>();
+
+		_serviceTracker = ServiceTrackerFactory.open(
```

Comment:
In the meantime we have a discussion with Mike Han and we ended up with the same solution, so I'll close this one for now and we move forward with this approach,

thank you for your time!
Daniel